Makefile

```diff
@@ -330,7 +330,10 @@ $(shell mkdir -p $(LOCAL_GOCACHE_PATH))
 endif
 
 .PHONY: build-images
-build-images: ## Build bpfman-agent and bpfman-operator images.
+build-images: build-bpfman-operator-image build-bpfman-agent-image ## Build bpfman-agent and bpfman-operator images.
+
+.PHONY: build-bpfman-operator-image
+build-bpfman-operator-image: ## Build bpfman-operator image.
```

Comment:
Do we need "bpfman" in the name? I like shorter names when they might be typed in.

Reply:
> Do we need "bpfman" in the name? I like shorter names when they might be typed in.

No, not really. We're implicitly in a "bpfman" repo.

```sh
% make help | grep -E '\w+build'
  build-release-yamls  Generate the crd install bundle for a specific release version.
  build            Build bpfman-operator and bpfman-agent binaries.
  build-images     Build bpfman-agent and bpfman-operator images.
  build-operator-image  Build bpfman-operator image.
  build-agent-image  Build bpfman-agent image.
```

Changed in https://github.com/bpfman/bpfman-operator/compare/ba4ba25cb525f8eabdaea316c89da835d0fbbbb1..bc5754662f96c389a86dca066c3768a6f5e0b56b.

Reply:
> ```shell
> % make help | grep -E '\w+build'
>   build-release-yamls  Generate the crd install bundle for a specific release version.
>   build            Build bpfman-operator and bpfman-agent binaries.
>   build-images     Build bpfman-agent and bpfman-operator images.
>   build-operator-image  Build bpfman-operator image.
>   build-agent-image  Build bpfman-agent image.

Should I drop "bpfman-" from the help text too?

Reply:
No, that's fine.